src/components/datetime/DateTimeController.cpp

```diff
@@ -46,7 +59,9 @@ void DateTime::SetTime(uint16_t year, uint8_t month, uint8_t day, uint8_t hour,
   UpdateTime(previousSystickCounter, true);
   xSemaphoreGive(mutex);
 
-  systemTask->PushMessage(System::Messages::OnNewTime);
+  if (systemTask) {
```

Comment:
The other checks in the file do `systemTask != nullptr`, maybe worth being consistent? Very minor though